src/porepy/numerics/fv/fvutils.py

```diff
@@ -1310,6 +1310,7 @@ def map_subgrid_to_grid(g, loc_faces, loc_cells, is_vector):
 def compute_darcy_flux(
     gb,
     keyword="flow",
+    keyword_store=None,
```

Comment:
I think we need a consistent way of doing this. Not this PR though, but something to think about

Reply:
I agree, this is rudimentary at the moment. We should also implement restart. I have some ideas about combining this with vtk output, but ideas are most welcome. Not in this PR, though.